ajax.php

```diff
@@ -50,3 +50,5 @@ function utf8_encode_recurse($mixed_value) {
         }
     }
 }
+
+require ('includes/application_bottom.php');
```

Comment:
Question: Doesn't the `exit()` on line 47 negate the calling of application_bottom?  

So, second question: is this actually fixing the original thing you intended it to?